flow360/component/flow360_params/flow360_params.py

```diff
@@ -1093,6 +1094,18 @@ def to_flow360_dict(self) -> dict:
         flow360_dict = json.loads(self.to_flow360_json())
         return flow360_dict
 
+    def to_flow360_json_file(self, filename: str) -> NoReturn:
+        """Exports :class:`Flow360Params` instance to .json file
+
+        Example
+        -------
+        >>> params.to_flow360_json_file() # doctest: +SKIP
+        """
+
+        flow360_dict = self.to_flow360_dict()
+        with open(filename, "w", encoding="utf-8") as fh:
+            json.dump(flow360_dict, fh, indent=4)
```

Comment:
I see now a little inconsistency with names. We have:
```python
.dict() -> dict
.json() -> str
```
Additionally we have functions to save file:
```python
.to_file(filename) -> None
.to_json(filename) -> None
```

So I think we should rename functions:
```python
.to_flow360_json() rename to .flow360_json() -> str
.to_flow360_dict() rename to .flow360_dict() -> dict
.to_flow360_json_file(filename) rename to .to_flow360_json(filename)
```

Reply:
fixed.